app/ui.js

```diff
@@ -8,18 +8,25 @@
  */
 
 /* jslint white: false, browser: true */
-/* global window, $D, Util, WebUtil, RFB, Display */
+/* global window, document.getElementById, Util, WebUtil, RFB, Display */
 
+/* [module] name: UI; requires: Util, WebUtil, KeyTable, RFB, Display */
 var UI;
 
 (function () {
     "use strict";
 
+    //var resizeTimeout;
```

Comment:
I assume this was a miss when rebasing, this variable has been moved inside the UI object.